policy/modules/services/git.te

```diff
@@ -92,12 +101,25 @@ type git_session_t, git_daemon;
 userdom_user_application_domain(git_session_t, gitd_exec_t)
 role git_session_roles types git_session_t;
 
+attribute git_client_domain;
+type git_exec_t;
+application_executable_file(git_exec_t)
+
 type git_sys_content_t;
 files_type(git_sys_content_t)
 
 type git_user_content_t;
 userdom_user_home_content(git_user_content_t)
 
+type git_home_t;
+userdom_user_home_content(git_home_t)
+
+type git_home_hook_t;
```

Comment:
Is there anywhere else the hook could be? If not, git_hook_t would be fine.

Reply:
I anticipate that in the future there may be a need for a hook type in system git repositories, that being a `git_sys_hook_t`.